\hypertarget{classpip_1_1__vendor_1_1urllib3_1_1exceptions_1_1_new_connection_error}{}\doxysection{pip.\+\_\+vendor.\+urllib3.\+exceptions.\+New\+Connection\+Error Class Reference}
\label{classpip_1_1__vendor_1_1urllib3_1_1exceptions_1_1_new_connection_error}\index{pip.\_vendor.urllib3.exceptions.NewConnectionError@{pip.\_vendor.urllib3.exceptions.NewConnectionError}}


Inheritance diagram for pip.\+\_\+vendor.\+urllib3.\+exceptions.\+New\+Connection\+Error\+:
% FIG 0


Collaboration diagram for pip.\+\_\+vendor.\+urllib3.\+exceptions.\+New\+Connection\+Error\+:
% FIG 1
\doxysubsection*{Additional Inherited Members}


\doxysubsection{Detailed Description}
\begin{DoxyVerb}Raised when we fail to establish a new connection. Usually ECONNREFUSED.\end{DoxyVerb}
 

The documentation for this class was generated from the following file\+:\begin{DoxyCompactItemize}
\item 
venv/\+Lib/site-\/packages/pip/\+\_\+vendor/urllib3/exceptions.\+py\end{DoxyCompactItemize}
